docs/LOCAL_DEPLOYMENT.md

```diff
@@ -91,14 +91,13 @@ This step is included if you cannot use the Launch configuration in VSCode. Open
 ```shell
 cd code
 python -m pip install -r requirements.txt
-cd app
 python -m flask --app ./app.py --debug run
```

Comment:
We also encountered this problem during local deployment. After preliminary investigation, it may be caused by changes in the directory structure. You just need to use `pip install InteractiveInterpreter`, reinstall this package. Of course, we will continue to follow up on this issue.

Reply:
This bug is caused by changes in the directory structure. Since the `app.py` file is currently in the root directory of the `code` folder, and the `werkzeug` package import the `code` in the system lib, it is considered to have been rewritten if it has the same name. I changed `code` folder to `codes` and this problem no longer exists.

Reply:
The above method has been abandoned. We have solved it in other ways. You can retest it, thanks!